weblate/templates/snippets/project/billing-libre-trial.html

```diff
@@ -8,6 +8,6 @@
   {% if user_can_view_billing %}
     <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for libre hosting" %}</a>
   {% endif %}
-  {% trans "This project is in the trial period, so be cautious before you contribute. Things like license or repository can still change before being approved as a libre project." %}
+  {% trans "This project is in a trial period and might not yet be fully set up! </br> Better ask its maintainers before you contribute to prevent the chance of losing your work. Also, things like license or repository can still change before being approved as a libre project or not." %}
```

Comment:
```suggestion
  {% trans "This project has neither been fully set up, nor accepted as a libre project.</br>Only contribute to it if you are OK with the particular libre license changing before then, and save your work so it is not lost." %}
```

Reply:
I took inspiration from your suggestion. I didn’t accept it as a whole because all the changes are optional. Got rid of the repository as, yes, translators don’t know about this even if that happens so let’s not bother them with it. :)

Reply:
resolving as outdated